thinkhazard/templates/report.jinja2

```diff
@@ -31,8 +31,20 @@ Think Hazard - {{ division.name}}
           <a href="">About</a>
         </li>
         <li class="disabled">
+          <a href="">FAQ</a>
+        </li>
+        <li class="disabled">
+          <a href="">References</a>
+        </li>
+        <li class="disabled">
           <a href="">Contact us</a>
         </li>
+        <li class="dropdown disabled">
+          <a href="" data-toggle="dropdown" class="dropdown-toggle">English <i class="fa fa-chevron-down"></i></a>
+          <ul class="dropdown-menu">
+            <li><a href="">English</a></li>
+          </ul>
+        </li>
```

Comment:
Since the link in the navbar are supposed to be the same in index and report pages, we should put this in a separate (included) template.